docs/websites/nginx/nginx-ssl-and-tls-deployment-best-practices.md

```diff
@@ -0,0 +1,286 @@
+---
+author: 
+  name: Linode Community
+  email: docs@linode.com
+description: 'This guide details best practices for deploying SSL and TLS in conjunction with nginx.'
+keywords: 'nginx,ssl,tls'
+license: '[CC BY-ND 4.0](http://creativecommons.org/licenses/by-nd/4.0)'
+published: 'Monday, July 11th, 2016'
+modified: Monday, July 11th, 2016
+modified_by:
+  name: Linode
+title: 'Nginx SSL and TLS Deployment Best Practices'
+contributor:
+  name: Ryan Laverdiere
+  link: https://github.com/capecodrailfan
+external_resources:
+- '[Qualys SSL Labs SSL Server Test](https://www.ssllabs.com/ssltest/)'
+- '[KeyCDN HTTP/2 Test](https://tools.keycdn.com/http2-test)'
+---
+
+*This is a Linode Community guide. [Write for us](/docs/contribute) and earn $250 per published guide.*
+<hr>
+
+This guide is intended to inform you of some additional configuration options that nginx uses when serving HTTPS. While these features help optimize nginx for SSL and TLS, this is by no means a complete guide to securing nginx or your Linode. The best way to ensure your server remains secure is to not only to configure it properly, but to follow best security practices at all times. This guide is intended to be one of many steps toward creating the most secure environment possible. 
+
+## Before you Begin
+
+1.  This guide is a continuation of our tutorial on how to [install nginx and a StartSSL certificate on Debian 8 (Jessie)](/docs/websites/nginx/install-nginx-and-a-startssl-certificate-on-debian-8-jessie). The principles here can be adapted to an SSL/TLS deployment on any system, but at a minimum, you will need a Linode with the latest stable version of nginx and an SSL certificate installed.
+
+2.  Update your system:
+
+        apt-get update && apt-get upgrade
+
+{: .note}
+>
+>The commands in this guide are written for a root user. If you're following along as a non-root user, commands that require elevated privileges should prefixed with `sudo`. If you’re not familiar with the `sudo` command, you can check our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
+
+For more information, please review our guides on [basic nginx configuration](/docs/websites/nginx/how-to-configure-nginx), [Linux security basics](/docs/security/linux-security-basics), and [securing your server](/docs/security/securing-your-server).
```

Comment:
Proof note: delete comma before "and" (Oxford comma not necessary when font alterations indicate disparate items in a series.)